EndpointHandler.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require "settings.php";
```

Comment:
Consider using `require_once` instead of `require`, in case it has already been `require`d: [http://stackoverflow.com/questions/2418473/when-should-i-use-require-once-vs-include](http://stackoverflow.com/questions/2418473/when-should-i-use-require-once-vs-include)